package.json

```diff
@@ -28,7 +28,9 @@
 	],
 	"scripts": {
 		"build": "tsup src/index.ts --clean --format cjs,esm --outDir lib --dts && cp lib/index.d.ts lib/index.d.cts",
-		"docs": "typedoc",
+		"docs": "pnpm run docs:build && pnpm run docs:build-post",
+		"docs:build": "typedoc",
+		"docs:build-post": "sh scripts/build-post.sh",
```

Comment:
`lint:spelling` (cspell) was failing on a couple of words in the script. I also don't like putting a lot of logic in `package.json`s - it gets unweildy. So I moved it to a separate file. WDYT @RebeccaStevens?

Reply:
`docs:build-post` no longer needed that to update in `typedoc-plugin-versions`